web/views/page_view.ex

```diff
@@ -59,5 +59,8 @@ defmodule Askbywho.PageView do
     |> raw
   end
 
-
+  def on_qa do
+    IO.puts("environment var is_qa is #{inspect System.get_env("is_qa")}")
+    System.get_env("is_qa") == "yes"
```

Comment:
initially was going to do that but now QA thinks it is prod so I would have needed to set up a new environment and seemed there would have been a lot of config to do, this was quicker